src/test/groovy/com/skalashynski/spring/springboot/web/controller/StudentControllerTest.groovy

```diff
@@ -35,153 +31,202 @@ class StudentControllerTest extends DatabaseSpecification {
+                    Student)
+        } catch (Exception ex){
+            actualErrorMessage = ex.getMessage();
+        }
         then:
-            List<Student> actualStudent = response.getBody()
-            assert actualStudent.size() == count
+        def actualStudent = response.getBody()
+        actualStudent == savedStudent
+        response.getStatusCode() == httpCode
+
         where:
-            name        || count
-            'Aliko'     || 1
-            'Bill'      || 2
-            'Folrunsho' || 3
-            'Dima'      || 0
+        student                                                                                              | httpCode        | savedStudent
+        new Student("Viktor", "Shilay", LocalDate.of(1999, 1, 22), LocalDateTime.of(2022, 8, 13, 9, 20))     | HttpStatus.OK   | new Student(7, "Viktor", "Shilay", LocalDate.of(1999, 1, 22), LocalDateTime.of(2022, 8, 13, 9, 20))
     }
 
-    @Unroll
-    def void "Delete student by #id, expected #code"(Long id) {
+    def "Students get all, expected 6"() {
         when:
-            def response
-            def actualErrorMessage
-            try {
-                response = restTemplate.exchange(
-                        STUDENT_URL + id
-                        , HttpMethod.DELETE
-                        , new HttpEntity<>(headers)
-                        , Void.class)
-            } catch (Exception e) {
-                actualErrorMessage = e.getMessage()
-            }
+        def response
+        def actualErrorMessage
+        try {
+            response = restTemplate.exchange(
+                    STUDENT_URL
+                    , HttpMethod.GET
+                    , new HttpEntity<>(headers)
+                    , List<Student>)
+        } catch (Exception e) {
+            actualErrorMessage = e.getMessage()
+        }
         then:
-            assert response.getStatusCode() == code
+        List<Student> actualStudents = response.getBody()
+        actualStudents.size() == 6
+    }
+
+    def "Students get by #firstName, expected #count"() {
+        when:
+        def response
+        def actualErrorMessage
+        try {
+            response = restTemplate.exchange(
+                    STUDENT_URL + "search?first_name=" + firstName
+                    , HttpMethod.GET
+                    , new HttpEntity<>(headers)
+                    , List<Student>)
+        } catch (Exception e) {
+            actualErrorMessage = e.getMessage()
+        }
+        then:
+        List<Student> actualStudent = response.getBody()
+        actualStudent.size() == count
         where:
-            id || code
-            1  || HttpStatus.NO_CONTENT
-            2  || HttpStatus.NO_CONTENT
-            3  || HttpStatus.NO_CONTENT
-            8  || HttpStatus.NOT_FOUND
+        firstName       || count
+        'Aliko'         || 1
+        'Bill'          || 2
+        'Folrunsho'     || 3
+        'Dima'          || 0
```

Comment:
please, configure your IDEA to add some spaces in this section. You are formatting already predefined code style.

Reply:
fixed